test/verify/check-machines-disks

```diff
@@ -847,6 +847,35 @@ class TestMachinesDisks(VirtualMachinesCase):
         b.wait_not_present("#vm-subVmTest1-disks-vdd-device")
         b.wait_not_present(".pf-c-modal-box")
 
+        # Test detaching several disks and the deletion dialog can be closed correctly
+        m.execute("virsh vol-create-as myPoolOne diskVirtio --capacity 1M --format qcow2")
+        m.execute("virsh vol-create-as myPoolOne diskSata --capacity 1M --format qcow2")
+        wait(lambda: "diskVirtio" in m.execute("virsh vol-list myPoolOne"))
+        wait(lambda: "diskSata" in m.execute("virsh vol-list myPoolOne"))
+        m.execute("virsh attach-disk --domain subVmTest1 --source {0}/diskVirtio --target vdf --targetbus virtio --persistent".format(p1))
+        m.execute("virsh attach-disk --domain subVmTest1 --source {0}/diskSata --target sda --targetbus sata --persistent".format(p1))
+
+        # Need to refresh when attaching disk for shutoff VM
+        b.reload()
+        b.enter_page('/machines')
+        b.wait_in_text("body", "Virtual machines")
+        b.wait_visible("#vm-subVmTest1-disks-sda-device")
+        b.wait_visible("#vm-subVmTest1-disks-vdf-device")
+
+        def removeDiskAndChecks(target):
+            b.click("#delete-subVmTest1-disk-{}".format(target))
+            b.wait_visible("#app + div div[role=dialog]")
+            b.wait_in_text("#app + div span", target)
```

Comment:
Please avoid using such selectors that contain 0 information for the developer about what element it's actually selecting. 
They have two main issue:

* really hard to read the test code
* if I change the code slightly to have another div this selector will break.

In this case you should rather add an #id in the js code to the item so that you can use that one. Same for the dialog element and the buttons.

Reply:
Sorry for not making it obvious in the first review round. I would really prefer to get this in a more readable state before merging.

Reply:
OK, but I would like to try to change the selector firstly. How do the new selectors look like?

Reply:
We will need to introduce something like this https://github.com/cockpit-project/cockpit/blob/master/pkg/machines/components/vm/disks/vmDisksCard.jsx#L289 for the resource modal component.

Then we can use  id={`delete-${objectId}-button`} for the remove button and  id={`delete-${objectId}-modal`} for the modal selector. 

I can also do it and push on your branch if you prefer.